packages/expo-notifications/android/src/main/java/expo/modules/notifications/service/interfaces/CategoriesDelegate.kt

```diff
@@ -0,0 +1,9 @@
+package expo.modules.notifications.service.interfaces
+
+import expo.modules.notifications.notifications.model.NotificationCategory
+
+interface CategoriesDelegate {
```

Comment:
```suggestion
interface NotificationsCategoriesDelegate {
```
🤔? Maybe a line of comment for this delegate as well?

Reply:
👍 I will add comment, but I will refrain from renaming the interface — I will do this in a separate "cleanup" PR after all the parts are extracted.